src/goMain.ts

```diff
@@ -434,6 +440,7 @@ function sendTelemetryEventForConfig(goConfig: vscode.WorkspaceConfiguration) {
 		includeImports: goConfig['gotoSymbol'] && goConfig['gotoSymbol']['includeImports'] + '',
 		addTags: JSON.stringify(goConfig['addTags']),
 		removeTags: JSON.stringify(goConfig['removeTags']),
+		fillStruct: JSON.stringify(goConfig['fillStruct']),
```

Comment:
This is only for configuration. Since there is no config called `fillStruct`, this change is not needed

Reply:
Done.